### PR TITLE
Fixed #26002 - Custom search operator example in get_search_results

### PR DESCRIPTION
**Pull request by bbenko**
Improved example and documentation in get_search_results.

### CODE REVIEW

**Comment by mjtamlyn** [comment]
This looks like you should be able to use it to use PGFTS as well? This would be a nice subsequent feature for contrib.postgres :)

**Comment by bbenko** [comment]
@mjtamlyn yes, let's do it. :)

**Comment by timgraham** [comment]
buildbot, add to whitelist.

File: docs/ref/contrib/admin/index.txt [hunk 1]
```diff
@@ -1381,25 +1381,41 @@ templates used by the :class:`ModelAdmin` views:
...
+                queryset, use_distinct = super(PersonAdmin, self).get_search_results(
+                    request, queryset, ' '.join(include_search_terms)
+                )
+
+                # Remove all search results we don't want.
+                if exclude_search_terms:
+                    exclude_queryset, _ = super(PersonAdmin, self).get_search_results(
+                        request, queryset, ' '.join(exclude_search_terms)
+                    )
+                    queryset = queryset.exclude(pk__in=exclude_queryset)
```
Comment (timgraham) [inline docs/ref/contrib/admin/index.txt]: I'm a bit nervous about recommending the use of `pk__in`. I think that doesn't scale very well if the list of IDs is huge.

**Comment by phildini** [comment]
@timgraham is the concern high enough that you think this patch needs revising, to use a different example?

**Comment by timgraham** [comment]
I haven't completely decided. If someone has another idea in mind that avoids the issue, that would certainly be useful to compare to.

**Comment by timgraham** [comment]
I submitted a simpler patch to explain the current example: https://github.com/django/django/pull/6844.

**Pull request closed without merging**